app/views/posts/show.html.erb

```diff
@@ -1 +1,15 @@
-<h1>Here is post <%=@post_id %> by user <%=@user %></h1>
\ No newline at end of file
+<div class="recent-posts">
+  <p><%= @post.text %></p>
+  <p class="counters">comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></p>
+
+  <% if @post.recent_comments.any? %>
+    <h3 class="comments-heading">Comments:</h3>
+    <ul class="comments-list">
+      <% @post.recent_comments.each do |comment| %>
+        <li>
+          <%= comment.user.name %> : <%= comment.text %>
+        </li>
+      <% end %>
+    </ul>
```

Comment:
- [ ] Great job so far @Ashnaali3255 There is just one small change I will suggest you Implement. Kindly ensure that when on the Posts Show page, the Page shows all the comments for the post. Currently, it only shows recent posts which are 5 instead of 6 comments I created for that post :wink: 
- Your Posts Show Page
![image](https://github.com/Ashnaali3255/Blog-website/assets/102699958/89574c15-8292-4c05-a809-c38c439b6f2f)
- The Given Wireframe
![image](https://github.com/Ashnaali3255/Blog-website/assets/102699958/9a21df88-e37c-4236-9628-95fe2d7a168c)